### PR TITLE
Inline the Draft2019-09 subschema definitions

### PR DESCRIPTION
**Pull request by tomdeering-wf**
## Ultimate problem:
Tests have issues retrieving referenced subschemas within the Draft 2019 definition

## How it was fixed:
Replace the refs in the schema by inlining the subschemas.

## Testing suggestions:
Passing CI

## Potential areas of regression:

---

> __FYA:__ @michaelcarter-wf

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: test/unit/constants.dart [hunk 1]
```diff
@@ -325,4 +324,10 @@ final List<String> draft9SkippedTestFiles = [
   "vocabulary.json",
 ]..addAll(commonSkippedTestFiles);
 
+final List<String> draft2019SkippedTests = [
+  'remote ref, containing refs itself : remote ref valid',
+  'remote ref, containing refs itself : remote ref invalid',
```
Comment (robbielamb-wf) [inline test/unit/constants.dart:329]: These 2 ref tests don't need to be skipped as this work goes in.

File: lib/src/json_schema/json_schema.dart [hunk 2]
```diff
@@ -590,7 +590,9 @@ class JsonSchema {
     // Fallback order for ref provider:
     // 1. Base URI (example: localhost:1234/integer.json)
     // 2. Base URI with empty fragment (example: localhost:1234/integer.json#)
-    final dynamic schemaDefinition = _refProvider.provide(baseUri.toString()) ?? _refProvider.provide('${baseUri}#');
+    final dynamic schemaDefinition = getJsonSchemaDefinitionByRef(ref.toString()) ??
+        _refProvider.provide(baseUri.toString()) ??
+        _refProvider.provide('${baseUri}#');
```
Comment (robbielamb-wf) [inline lib/src/json_schema/json_schema.dart:595]: Should this comment be updated?

File: test/unit/constants.dart [hunk 3]
```diff
@@ -310,9 +310,6 @@ final List<String> draft2019SkippedTestFiles = [
 ]..addAll(commonSkippedTestFiles);
 
 final List<String> draft2019SkippedTests = [
-  // Requires loading draft 2019 as a remote reference.
-  'remote ref, containing refs itself : remote ref valid',
-  'remote ref, containing refs itself : remote ref invalid',
```
Comment (robbielamb-wf) [inline test/unit/constants.dart:315]: :tada:

**Review by robbielamb-wf — APPROVED** [review]

**Pull request merged**